jetstream/workflows/run.yaml

```diff
@@ -10,13 +10,22 @@ spec:
     - name: project_id
     - name: dataset_id
     - name: bucket
+    - name: enrollments
+    - name: recreate_enrollments
   templates:
   - name: jetstream
     parallelism: 5  # run up to 5 containers in parallel at the same time
     inputs:
       parameters:
         - name: experiments
     steps:
+    - - name: ensure-enrollments
+        template: ensure_enrollments
+        arguments:
+          parameters:
+          - name: experiment
+            value: "{{item}}"
+        withParam: "{{input.parameters.enrollments}}"
```

Comment:
This step will create enrollment tables for all experiments in the analysis run. If one of these steps would fail for some reason (BigQuery SQL error, ...), then also all future steps will fail. Unfortunately, Argo doesn't support [`dependencies`](https://argoproj.github.io/argo/enhanced-depends-logic/) in loops